src/App.jsx

```diff
@@ -1,18 +1,21 @@
 import React from "react";
 import Table from "./Table";
+/*Add import statement here*/
```

Comment:
./src/App.jsx
  Line 2:  'Table' is defined but never used  no-unused-vars
  Line 3:  'List' is defined but never used   no-unused-vars

Search for the keywords to learn more about each warning.
To ignore, add // eslint-disable-next-line to the line before.